### PR TITLE
Add name to mentor list

### PR DESCRIPTION
**Pull request by rajansandeep**
Adding my name to the mentor list for the Kubernetes Cluster Addon project.

### CODE REVIEW

**Comment by idvoretskyi** [comment]
@justinsb can you please confirm that you are fine with this update?

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ _Add your project ideas below:_
   and plan to get involved with the kubernetes community in fixing
   them.
 - Recommended Skills: Go, Kubernetes
-- Mentor(s): Justin Santa Barbara (@justinsb), Leigh Capili (@stealthybox)
+- Mentor(s): Justin Santa Barbara (@justinsb), Leigh Capili (@stealthybox), Sandeep Rajan (@rajansandeep)
```
Comment (idvoretskyi) [inline README.md]: One of the subproject maintainers (eg. @justinsb) has to approve this.

Comment (rajansandeep) [reply README.md]: I discussed this with @stealthybox, who is one of the maintainers and mentor for this project

Comment (idvoretskyi) [reply README.md]: Great, so please let @stealthybox approve this.

Comment (stealthybox) [reply README.md]: +1

**Comment by idvoretskyi** [comment]
Thanks @stealthybox

**Comment by justinsb** [comment]
+1 from me also - sorry for delay and great to have you on board @rajansandeep :-)

**Pull request merged**